packages/assets-controllers/src/NftController.ts

```diff
@@ -143,6 +143,7 @@ export interface NftMetadata {
   lastSale?: ApiNftLastSale;
   transactionId?: string;
   tokenURI?: string | null;
+  error?: string | undefined;
```

Comment:
What is this property used for? If it's an error for use to use for diagnostics, we typically use the `Error` type instead (which includes a stack trace). Though if that's what this is, I'm not sure why it would be included in NFT metadata.

If it's meant to represent an "error status", maybe an enum would be better suited here? We certainly wouldn't want to display a string directly anywhere, since it's not localized.

Reply:
The changes look good here! Thanks for sharing that link as well